.travis.yml

```diff
@@ -123,12 +123,11 @@ jobs:
         RUN_TESTS=true
         GOAL="install"
         BITCOIN_CONFIG="--enable-zmq --with-incompatible-bdb --enable-glibc-back-compat --enable-reduce-exports --with-gui=qt5 CPPFLAGS=-DDEBUG_LOCKORDER"
-# x86_64 Linux, No wallet
+# x86_64 Linux
```

Comment:
Is this change intentional? I find it very handy to have a NO_WALLET build available. It has happened to me before that I changed something in a way which required the wallet in a place which does not require a wallet overall. This particular travis build caught that.

Reply:
Sorry, I just noticed your description does mention that. So it is intentional. My comment still stands though, I find it beneficial to have a no-wallet build available. In a perfect world a build would build all possible combinations of build flags, in this imperfect world it can at least check the no-wallet case.

Reply:
Indeed, will undo this.